src/webui/src/components/PackageSidebar/index.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import API from '../../../utils/api';
+
+import classes from './style.scss';
+import getRegistryURL from '../../../utils/getRegistryURL';
+
+export default class PackageSidebar extends React.Component {
+  state = {
+    lastUpdate: 'Loading',
+    recentReleases: [],
+    author: null,
+    contributors: null,
+    showAllContributors: false,
+    dependencies: {}
+  };
+
+  constructor(props) {
+    super(props);
+    this.showAllContributors = this.showAllContributors.bind(this);
```

Comment:
You can use here https://babeljs.io/docs/plugins/transform-class-properties/ in the `jests` branch I already installed such dependency. That will avoid using the bind.

Reply:
@juanpicado It will break `no-invalid-this` rule in eslint, or it's fixed in latest eslint release?